x-pack/plugin/relevance-search/src/test/java/org/elasticsearch/xpack/relevancesearch/relevance/boosts/ScriptScoreBoostsTests.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.relevancesearch.relevance.boosts;
+
+import org.elasticsearch.test.ESTestCase;
+
+public class ScriptScoreBoostsTests extends ESTestCase {
+    public void testValueSource() {
```

Comment:
Do we want all of our asserts in the same test method, or should each individual assert be its own test method? Not sure what the best practices here are.

Reply:
I've seen tests in codebase that do it this way. I think if it's a lot of tiny similar asserts grouped together, it's ok not to write a tiny little method for each. Shorter code his way.

Reply:
I don't have strong feelings about this, so I'm OK leaving as is. 🙂